src/ots.h

```diff
@@ -232,6 +232,9 @@ bool IsValidVersionTag(uint32_t tag);
 #define OTS_TAG_VVAR OTS_TAG('V','V','A','R')
 #define OTS_TAG_STAT OTS_TAG('S','T','A','T')
 
+#define OTS_MAX_DECOMP_FILE_SIZE 300 * 1024 * 1024
```

Comment:
Please add a comment with a link to the GitHub issue above so the next time someone wonders about the arbitrary limit can easily find some answer.

Reply:
Also, please use the full word for the macro, `OTS_MAX_DECOMPRESSED_FILE_SIZE`.

Reply:
done in https://github.com/khaledhosny/ots/pull/229/commits/f21ab4ef2ff0d7c90ed2282341098b7354cbe5a4